File: Vocab/Vocab/vocab.cpp
#include "vocab.h"
#include <string>
#include <sstream>
#include <vector>

using std::string;

vocab::vocab(string english, string german)
{
	this->english = english;
	this->german = german;
}
string vocab::getGerman()
{
	return german;
}

string vocab::getEnglish()
{
	return english;
}

bool vocab::compareGerman(string german)
{
	return (german.compare(this->german) == 0);
}

string vocab::toString()
{
	return english + " " + german;
}

File: Vocab/Vocab/vocab.cs
namespace Vocab
{
    class vocab
    {
        private string english;
        public string German { get; private set; }

        public string English { get { return english; } private set { english = value; } }

        public vocab(string english, string german)
        {
            this.English = english;
            this.German = german;
        }

        public bool compareGerman(string german)
        {
            return German == german;
        }

        public override string ToString()
        {
            return English + " " + German;
        }
    }
}
